### PR TITLE
Copy update on /security

### LINKED ISSUE #11042: [COPY UPDATE] index - "Security"
Please update index - "Security" from the [copy doc](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Copy update on `/security`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit#) and resolve comments

## Issue / Card

Fixes #11042 

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/security/index.html [hunk 1]
```diff
@@ -73,28 +73,27 @@ <h2>
...
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/fd84bbdc-Document-open.svg",
-              alt="",
-              height="47",
-              width="56",
-              hi_def=True,
-              loading="lazy"
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
```
Comment (ClementChaumel) [inline templates/security/index.html:85]:
```suggestion
          {{
            image(
            url="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg",
```

File: templates/security/index.html [hunk 2]
```diff
@@ -73,28 +73,27 @@ <h2>
...
+            alt="",
+            width="43",
+            height="50",
```
Comment (ClementChaumel) [inline templates/security/index.html:88]:
```suggestion
            width="56",
            height="49",
```

**Review by ClementChaumel — COMMENTED** [review]
The copy changes look good but the datasheet image is a bit shrank compared to the original
image

vs 

image


I've suggested some values that _might_ fix the issue.

**Comment by minkyngkm** [comment]
Thanks @ClementChaumel for your suggestion! :) I've made some updates. Hope it looks better now!

**Review by ClementChaumel — APPROVED** [review]
Much better thanks for updating! :)

**Pull request merged**